index.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const Clubhouse = require('clubhouse-lib')
+const { ArgumentParser } = require('argparse')
+const { Octokit } = require("@octokit/rest")
+
+let parser = new ArgumentParser({
+  version: '1.0.0',
+  addHelp: true,
+  description: 'Tilt exterminator tool'
+});
+
+let subparsers = parser.addSubparsers()
+let syncParser = subparsers.addParser('sync', {
+  addHelp: true,
+  help: 'sync an issue from github to clubhouse',
+})
+
+syncParser.addArgument(['-i', '--issue'], {
+  help: 'The github issue number to import into clubhouse',
+  required: true,
+  type: Number,
+})
+
+let args = parser.parseArgs()
+let issueNumber = args.issue
+if (isNaN(issueNumber)) {
+  console.error(`error: --issue must specify a number`)
+  process.exit(1)
+}
+
+const clubhouseToken = process.env.CLUBHOUSE_API_TOKEN
+if (!clubhouseToken) {
+  console.error('Please set the CLUBHOUSE_API_TOKEN env variable')
+  process.exit(1)
+}
+
+const githubToken = process.env.GITHUB_API_TOKEN
+const octokitOptions = {}
+if (githubToken) {
+  octokitOptions.auth = githubToken
+} else {
+  console.warn('Warning: A GITHUB_API_TOKEN set in your environment will prevent you from getting rate-limited')
```

Comment:
nit: direction of this warning is unclear. Maybe consider:

> Warning: no GITHUB_API_TOKEN found in your env. Consider setting one to keep you from getting rate-limited.

Reply:
done!